### PR TITLE
RTVI 0.2.1

### PR DESCRIPTION
**Pull request by jptaylor**
- Daily package moved to `@daily-co/realtime-ai-daily`
- Fixed deprecation warning for services map

### CODE REVIEW

File: rtvi-client-js/src/client.ts [hunk 1]
```diff
@@ -165,21 +165,19 @@ export type RTVIEventCallbacks = Partial<{
...
-  onBotLlmStarted: (participant: Participant) => void;
-  onBotLlmStopped: (participant: Participant) => void;
-
-  onBotTtsText: (text: TTSTextData) => void;
-  onBotTtsStarted: (participant: Participant) => void;
-  onBotTtsStopped: (participant: Participant) => void;
+  onBotTranscript: (data: BotLLMTextData) => void;
+  onBotLlmText: (data: BotLLMTextData) => void;
+  onBotLlmStarted: () => void;
+  onBotLlmStopped: () => void;
```
Comment (aconchillo) [inline rtvi-client-js/src/client.ts:177]: What is the `participant` in `onBotStartedSPeaking`? Should that be removed as well?

Comment (jptaylor) [reply rtvi-client-js/src/client.ts:177]: @aconchillo you might be right, but I thought we did receive a participant object there. Let me check and update accordingly.

Comment (jptaylor) [reply rtvi-client-js/src/client.ts:177]: Confirmed these are not implemented. Removed.

**Review by markbackman — APPROVED** [review]
LGTM!

**Pull request merged**